langkit/metrics/themes/themes.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
I think we should simplify this module and just convert this into a python dict in a python file, don't worry about any of the json stuff. All of that will be handled in the policy in a single spot.